app/streaming/video/ffmpeg-renderers/renderer.h

```diff
@@ -68,4 +85,25 @@ class IFFmpegRenderer : public Overlay::IOverlayRenderer {
     virtual void notifyOverlayUpdated(Overlay::OverlayType) override {
         // Nothing
     }
+
+#ifdef HAVE_EGL
+    // By default we can't do EGL
+    virtual bool canExportEGL() {
+        return false;
+    }
+
+    virtual bool initializeEGL(EGLDisplay,
+                               const EGLExtensions &) {
+        return false;
+    }
+
+    virtual ssize_t exportEGLImages(AVFrame *,
+                                    EGLDisplay,
+                                    EGLImage[EGL_MAX_PLANES]) {
+        return -1;
+    }
+
+    // Free the ressources allocated during the last `exportEGLImages` call
+    virtual void freeEGLImages(EGLDisplay) {}
```

Comment:
I think the caller should need to provide the `EGLImage` array. That way the backend renderer doesn't have to remember them itself.

Reply:
You are right, I did this because the backend renderer is the owner of the frames, so in my mind it made sense that it would remember them